Vagrantfile

```diff
@@ -0,0 +1,14 @@
+Vagrant.configure("2") do |config|
+  
+config.vm.box = "fedora/32-cloud-base"
```

Comment:
Now I can see that the indentation is off. Please fix it.

Reply:
i think i fixed it